### DISCUSSION TITLE
Back to top button switched on but not shown in preview

### DISCUSSION THREAD

**Discussion by RSWEdev** [GetPublii/Publii#1806] [General] [answered]
Just created a new site with Easy Blog them 2.0 and wondering, why the back to top button is not shown at all.
Configured a page as homepage and do have multiple posts as well.
Any help?

Publii Version: 0.46.2 (build 16968)

**Comment by bobmitro** [comment, marked as answer]
Let's see it in action.

**Reply by RSWEdev** [reply]
top of lengthy page:
![image](https://github.com/user-attachments/assets/a0bf79a4-0f21-4d5b-b345-4400e359651e)

Bottom:
![image](https://github.com/user-attachments/assets/5b174311-708f-45f0-bef7-f4775d139bc7)

Site setting:
![image](https://github.com/user-attachments/assets/e39b0790-ef2f-48ff-b1d0-5e48588b3a15)

**Comment by bobmitro** [comment]
URL to the live site, please.

**Reply by RSWEdev** [reply]
Not (yet) live....
